### DISCUSSION TITLE
Getting an error: Missing argument 'MODE:[past|live]' after deploying to Heroku

### DISCUSSION THREAD

**Discussion by dstar22** [aahnik/tgcf#78] [Q&A] [answered]
Error: Missing argument 'MODE:[past|live]'.  Choose from:
2021-04-24T05:35:28.017627+00:00 app[worker.1]: 	past,
2021-04-24T05:35:28.017627+00:00 app[worker.1]: 	live.

getting this error on heroku 
![image](https://user-images.githubusercontent.com/66094119/115948750-a78a5f80-a4ed-11eb-9380-63b71f364565.png)

**Comment by aahnik** [comment, marked as answer]
in the config vars, set the `TGCF_MODE` to be live.

make a fresh deployment. did you deploy yesterday? it had some errors.

delete the app, and deploy a fresh one.
